@here/olp-sdk-dataservice-read/lib/index.ts

```diff
@@ -20,6 +20,7 @@
 export * from "./utils";
 export * from "./client";
 export * from "./cache";
+export * from "./layer";
```

Comment:
please, rename the folder to the "layerDataProvider"

Reply:
Done